nearswap/src/pool.rs

```diff
@@ -41,6 +42,16 @@ pub struct Pool {
     pub shares: LookupMap<AccountId, Balance>,
     /// check `PoolInfo.total_shares`
     pub total_shares: Balance,
+
+    populated: usize,
```

Comment:
please add comments what's populated and what's pivoted.

Reply:
Maybe we should rename populated to -> index?

Reply:
ok, adding comments, for `index` we have `last_updated_index`